ansible/localhost/virtualhost.conf

```diff
@@ -0,0 +1,24 @@
+<IfModule mod_ssl.c>
+
+<VirtualHost *:{{ http_port }}>
+    ServerAdmin webmaster@{{ domain }}
+    ServerName {{ domain }}
+    ServerAlias www.{{ domain }}
+    WSGIScriptAlias / /vagrant/holocron_api/holocron_api/wsgi.py
```

Comment:
I'm not going to block the PR for this, but we really should be hosting the "app" from inside a folder other than /vagrant

In prod, there won't be a /vagrant folder ... this should be configurable in some way. Not sure yet, something to think about for later.